docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,125 @@
+# Adding a New Metrics Provider for Dummy Endpoint
```

Comment:
to have the file show up on the docs site, you'll need to add it to the `nav` section in the `mkdocs.yml` config file at the root of the repo

Reply:
Alright let me do that.